index.html

```diff
@@ -0,0 +1,147 @@
+                hesitate to contact me.
+            </p>
+        </div>
+
+        <hr class="main-separator">
+
+        <section class="personel-feature-container">
+            <div class="feature-container">
+                <h4>Language</h4>
+                <ul>
+                    <li>
+                        <p>JavaScript</p>
+                    </li>
+                    <li>
+                        <p>Ruby</p>
+                    </li>
+                    <li>
+                        <p>Html</p>
+                    </li>
+                    <li>
+                        <p>Css</p>
+                    </li>
+                </ul>
+            </div>
+            <div class="feature-container">
+                <h4>Frameworks</h4>
+                <ul>
+                    <li>
+                        <p>Bootstrap</p>
+                    </li>
+                    <li>
+                        <p>Ruby on Rails</p>
+                    </li>
+                    <li>
+                        <p>RSPec</p>
+                    </li>
+                    <li>
+                        <p>CapyBara</p>
+                    </li>
+                    <li>
+                        <p>Selenium</p>
+                    </li>
+                </ul>
+            </div>
+            <div class="feature-container">
+                <h4>Skills</h4>
+                <ul>
+                    <li>
+                        <p>Codekit</p>
+                    </li>
+                    <li>
+                        <p>Github</p>
+                    </li>
+                    <li>
+                        <p>Codepen</p>
+                    </li>
+                    <li>
+                        <p>Gitlab</p>
+                    </li>
+                    <li>
+                        <p>Terminal</p>
+                    </li>
+                </ul>
+            </div>
+        </section>
+
+        <hr class="main-separator feature-icon">
+
+        <section class="social-media-container">
+            <div class="social-media">
+                <a href="#">
+                    <img src="assets/github.png" alt="github.com logo">
+                </a>
+                <a href="#">
+                    <img src="assets/linkedin.png" alt="linkedin.com logo">
+                </a>
+                <a href="#">
+                    <img src="assets/angellist.png" alt="angellist logo">
+                </a>
+                <a href="#">
+                    <img src="assets/twitter.png" alt="Twitter Logo">
+                </a>
+                <a href="#">
+                    <img src="assets/medium.png" alt="Medium.com logo">
+                </a>
+            </div>
+            <div class="work-arrow-container">
+                <a href="#">
+                    <img src="assets/arrow.png" alt="Arrow shows bottom">
+                    <p>Work</p>
+                </a>
+
+            </div>
+        </section>
```

Comment:
- [OPTIONAL]: Please convert this list of icons to a `<ul>` and `<li>` structure instead, all the icons should be then wrapped by an `<a>` tag as they currently are, you are only missing to switch the divs for the correct ul and li structure, here is a supporting article on how to use the structure correctly: [ul and li tags](https://www.w3schools.com/tags/tag_ul.asp)